packages/bpk-scrim-utils/src/bpk-scrim-content.scss

```diff
@@ -25,26 +25,10 @@
   bottom: 0;
   left: 0;
   z-index: $bpk-zindex-scrim;
-  overflow: auto;
```

Comment:
Why is this removed?

Reply:
overflow-x and overflow-y on the next two lines overwrote it.

Reply:
Can we keep it in for now? I remember explicitly adding this for x-browser compat - I should have added a comment explaining why :(